### PR TITLE
docs: create initial documentation 

### LINKED ISSUE #224: Create initial documentation
Take a look at e.g. https://geopandas.org/en/stable/getting_started.html

Suggested agenda:
- [x] 1. Introduction to SRAI (what is this)
- [ ] 2. Use cases of why to use srai 
- [x] 3. Getting Started / installation
- [x] 4. Send user to examples
- [x] 5. For full pipeline usage (for now) redirect user to https://github.com/Calychas/highway2vec_remaster/

Other tasks:
- [x] Swap API and Examples

### PR DESCRIPTION
**Pull request by piotrgramacki**
Closes #224 and #232

### CODE REVIEW

File: docs/getting_started/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Optional dependencies
+
+The following optional dependencies can be installed to enable additional functionality:
+
+* `srai[all]` - all optional dependencies
+* `srai[osm]` - dependencies required to download OpenStreetMap data
+* `srai[voronoi]` - dependencies to use Voronoi-based regionization method
+* `srai[gtfs]` - dependencies to process GTFS data
+* `srai[plotitng]` - dependencies to plot graphs and maps
```
Comment (RaczeQ) [inline docs/getting_started/README.md:21]: typo
```suggestion
* `srai[plotting]` - dependencies to plot graphs and maps
```

Comment (piotrgramacki) [reply docs/getting_started/README.md:21]: So typical of me ;)

**Review by Calychas — APPROVED** [review]

**Review by Calychas — DISMISSED** [review]
Great work!

File: docs/getting_started/use_cases.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We present use-cases in a form of user stories obtained using design thinking approach.
+
+## Pre-calculated embeddings
+
+> 🏗️ In progress... 🏗️
+>
+> Embeddings store is not yet available, as it introduces infrastructure costs to host them. We intend to release pre-trained embeddings from our methods in
+
+As a city planner, I would like to anticipate the amount of traffic on the estate. I work for one of the major cities, for which I found that there are pre-trained embeddings by the `srai` team. I want it to run fast and that I don't have to do anything other than download the embeds and a library that can read them.
```
Comment (Calychas) [inline docs/getting_started/use_cases.md:13]:
```suggestion
As a city planner, I would like to anticipate the amount of traffic on the estate. I work for one of the major cities, for which I found that there are pre-trained embeddings by the `srai` team (or by a member of the community). I want it to run fast and that I don't have to do anything other than download the embeds and a library that can read them.
```

File: docs/getting_started/use_cases.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Embedding training
+
+I work for a logistics company and want to use hex2vec embeddings to predict the demand in given aread of a city. I want them to be based only on our area of operations, so I need to train them myself. I need a simple solution to train my own hex2vec based on OSM data.
+
+## New data source
+
+> 🏗️ In progress... 🏗️
+>
+> Pre-trained model's weights are not yet available for download. We plan to reproduce our experimetns using `srai` and release weights for all of our published models
```
Comment (Calychas) [inline docs/getting_started/use_cases.md:31]: Duplicated from Transfer learning

**Review by Calychas — CHANGES_REQUESTED** [review]

File: docs/getting_started/use_cases.md (hunk 3 above)
Comment (piotrgramacki) [reply docs/getting_started/use_cases.md:31]: Yes, but both of these use cases are blocked by the same thing. I think it is ok to repeat it here as a reference to a a single use case

**Review by Calychas — DISMISSED** [review]

**Pull request merged**